src/components/CommentReplyPreview/CommentReplyPreview.tsx

```diff
@@ -98,30 +113,53 @@ export const CommentReplyPreview = ({
         </span>
       </div>
       {displayReplyComment && (
-        <div className={previewStyle}>
-          <p className={replyPreviewHeaderStyle}>
-            {commentBeingRepliedTo.userProfile.displayName} @{" "}
-            {commentBeingRepliedTo.date} said:
-          </p>
-          <p
-            dangerouslySetInnerHTML={{
-              __html: commentBeingRepliedTo.body || ""
-            }}
-          />
-          <div>
-            <span
-              className={cx(
-                hideCommentButtonStyles,
-                commentControlsButtonStyles,
-                previewHideCommentButtonStyles
-              )}
-              onClick={() => setDisplayReplyComment(!displayReplyComment)}
-            >
-              {displayReplyComment ? "Hide Comment" : "Show comment"}
-            </span>
-          </div>
-        </div>
+        <Preview
+          pillar={pillar}
+          commentBeingRepliedTo={commentBeingRepliedTo}
+          setDisplayReplyComment={setDisplayReplyComment}
+          displayReplyComment={displayReplyComment}
+        />
       )}
     </>
   );
 };
+
+export const Preview = ({
+  commentBeingRepliedTo,
+  pillar,
+  setDisplayReplyComment,
+  displayReplyComment
+}: {
+  commentBeingRepliedTo: CommentType;
+  pillar: Pillar;
+  setDisplayReplyComment: (displayReplyComment: boolean) => void;
+  displayReplyComment: boolean;
+}) => {
+  const commentControlsButtonStyles = commentControlsButton(pillar);
```

Comment:
Button should not coloured by pillar

![Screenshot 2020-04-02 at 18 38 46](https://user-images.githubusercontent.com/1336821/78280601-46be6c80-7511-11ea-8b55-508dcb29ccea.jpg)